projects/aca-content/src/lib/ui/variables/variables.scss

```diff
@@ -41,6 +41,8 @@ $black-heading: #4e4c4c;
 $grey-dropdown-background: #eee;
 $grey-divider: rgba(0,0,0,.22);
 $pagination-background-color: #ffffff;
+$black-87-alpha: rgba(black, 0.87);
```

Comment:
should be something domain, like the one below

Reply:
Updated the variable name. New variable name is `datetimepicker-font-color`